celery/loaders/base.py

```diff
@@ -89,6 +89,9 @@ def import_task_module(self, module):
         return self.import_from_cwd(module)
 
     def import_module(self, module, package=None):
+        want_reload_module = sys.modules.get(module, None)
```

Comment:
Why would we always want to reload already imported modules?

Reply:
I do not have a good explanation *why* except that it works for me. You can see from the test cases that it does not work *without* reloading. I would appreciate any hint why this issue occurs is at all.